### PR TITLE
My materials permission change, Photo fix

### PR DESCRIPTION
**Pull request by RomanPrigorodov**

### CODE REVIEW

**Comment by ShishckovA** [comment]
Not updated with master

File: front-end/src/components/Personnel/Student/StudentGeneral/StudentGeneral.vue [hunk 1]
```diff
@@ -285,6 +288,11 @@ export default {
   },
   async created() {
     await this.fetchInfo();
+    if (parseInt(this.$route.params.studentId, 16) !== this.userId) {
```
Comment (ShishckovA) [inline front-end/src/components/Personnel/Student/StudentGeneral/StudentGeneral.vue:291]: !!

File: front-end/src/components/Personnel/Teacher/Teacher.vue [hunk 2]
```diff
@@ -379,6 +382,11 @@ export default {
   },
   async created() {
     await this.fetchInfo();
+    if (parseInt(this.$route.params.teacherId, 16) !== this.userId) {
```
Comment (ShishckovA) [inline front-end/src/components/Personnel/Teacher/Teacher.vue:385]: !!

File: front-end/src/components/Schedule/Schedule.vue [hunk 3]
```diff
@@ -317,10 +317,10 @@ export default {
...
-  async created() {
-    this.filter.mg = this.milgroups[0]?.id.toString();
-    await this.fetchData();
-  },
+  // async created() {
+  //   this.filter.mg = this.milgroups[0]?.id.toString();
+  //   await this.fetchData();
+  // },
 
   methods: {
```
Comment (ShishckovA) [inline front-end/src/components/Schedule/Schedule.vue:325]: Let's search about store waiting

**Review by ShishckovA — COMMENTED** [review]
Some fikes required

**Review by deborisov — APPROVED** [review]

**Pull request merged**